ci/container/hang_webserver.py

```diff
@@ -20,6 +24,40 @@ def do_POST(self):
             self.send_response(200, 'OK')
             self.send_header('Content-Type', 'text/plain')
             self.end_headers()
+        elif self.path.startswith('/503'):
+            self.send_response(503)
+            self.send_header('Content-Type', 'text/plain')
+            self.end_headers()
+        elif self.path.startswith('/xml'):
+            self.send_response(200, '<error/>')
+            self.send_header('Content-Type', 'application/xml')
+            self.end_headers()
+        elif self.path.startswith('/resetCounter'):
+            HTTPRequestHandler.counter = 0
+            self.send_response(200, 'OK')
+            self.send_header('Content-Type', 'text/plain')
+            self.end_headers()
+        elif self.path.startswith('/eachThirdReturns200Others503'):
```

Comment:
Could you please give details about "/eachThirdReturns200Others503"?

Reply:
I simplified hang webserver and added description for what do I need this endpoint